frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/RecentlyUsedConstraints/useRecentlyUsedConstraints.test.tsx

```diff
@@ -91,26 +91,22 @@ describe('useRecentlyUsedConstraints', () => {
     });
 
     it('should initialize with empty array when no items in localStorage', () => {
-        const { result } = renderHook(() =>
-            useRecentlyUsedConstraints('test-key'),
-        );
+        const { result } = renderHook(() => useRecentlyUsedConstraints());
```

Comment:
useRecentlyUsedConstraints had key removed, as at first I wanted to make generic one, but now its specific to constraints, so it does not need key.